web-client/src/app/views/landing/landing.page.html

```diff
@@ -1,5 +1,5 @@
-<ion-content color="prussian-blue">
-  <ion-grid class="h-full sm:max-w-max">
+<ion-content>
+  <ion-grid class="h-full">
```

Comment:
This won't be necessary and will probably need to be removed when #23 get merged because the content color is set globally & the `max-width` is also being applied on `router-outlet` level

Reply:
Right, #23 will change this, but we should still merge a version with the right color in this PR, so that the baseline snapshots in Chromatic looks correct again. (The baseline snapshot is what all the following PRs will be compared with, so it's worth getting right, so that the UI tests can catch unintentional regressions.)

Does that sound good?

Reply:
Then let's merge #23 first so we have to change this one

Reply:
Cool, sounds good.